### ISSUE TITLE
Add a plugin icon

### ISSUE THREAD

**Issue by willmot** [humanmade/mad-mimi-sign-up-forms-wordpress-plugin#1]
Would be nice to have a plugin icon so that we look nice in search results.

![screen shot 2015-05-21 at 18 07 30](https://cloud.githubusercontent.com/assets/308507/7754409/419d9da4-ffe4-11e4-804f-1a0ac13d8107.png)

**Comment by Japh** [comment]
No `assets` directory in this repo for them to go, but you can use these if you like:

![Icon 128x128](https://dl.dropboxusercontent.com/u/774854/mad-mimi-assets/icon-128x128.png)

![Icon 256x256](https://dl.dropboxusercontent.com/u/774854/mad-mimi-assets/icon-256x256.png)

![Banner 772x250](https://dl.dropboxusercontent.com/u/774854/mad-mimi-assets/banner-772x250.png)

![Banner 1544x500](https://dl.dropboxusercontent.com/u/774854/mad-mimi-assets/banner-1544x500.png)

**Comment by willmot** [comment]
Thanks @Japh!

**Comment by willmot** [comment]
@saschamt @susandyoung are you happy to go with the above icons for this existing version? Also happy for you to provide icons and an updated header graphic if you'd like it to use this as an opportunity to change them.

**Comment by susandyoung** [comment]
I believe these are the most up to date. @saschamt can you confirm?

**Comment by saschamt** [comment]
There are simpler icons posted on the other thread. I had also said to provide new banner images tomorrow

**Comment by willmot** [comment]
Thanks @saschamt I'll replace these with the images uploaded to https://github.com/humanmade/godaddy-forms-wordpress-plugin/issues/2#issuecomment-108222097

**Comment by willmot** [comment]
waiting on getting SVN commit access.

**Comment by willmot** [comment]
These are now live!

https://wordpress.org/plugins/search.php?q=mad+mimi

**Comment by saschamt** [comment]
Very nice Tom, thank you!
